dimsim/tests/test_dimsim.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+# Standard libs
+import io
+import os
+# Dependencies
+import pytest
+
+# The module to test
+from dimsim.core.dimsim import get_distance, getCandidates
+
+def test_distance_near():
+        dist = get_distance('大侠','大虾')
+        assert dist == 0.0002380952380952381
+
+def test_distance_far():
+        dist = get_distance('大侠','大人')
+        assert dist == 25.001417183349876
+
+def test_get_candidates_simplified():
```

Comment:
Thanks @kmh4321 , I think we should add a test with a single `atom` for distance and candidates.